cmd/run.go

```diff
@@ -60,11 +60,11 @@ var runCommand = &cobra.Command{
 				volume = user.HomeDir + volume[1:]
 			}
 			dockerArgs = append(dockerArgs, "-v")
-			dockerArgs = append(dockerArgs, volume)
+			dockerArgs = append(dockerArgs, os.ExpandEnv(volume))
 		}
```

Comment:
@tjamet I dont use ``["KEY=value"]`` format in the docker file instead I use it in the package file generated by whalebrew upon installation.

I use this to preset run time values instead of creating a config file or having to set env variables every time the installed app is run.  Makes life easier. 

I use this method (editing the generated package file) for everything from mount paths, ports to env variables. Gave a few examples in #9. Hope that explains my use case.

As for expanding, I still dont get the why part of it.
Considering, ``LABEL io.whalebrew.config.environment '["key=${foo}","${foobar}","${bar}=value"]'`` the following are the cases I can think of:
1. for ``key=${foo}``to work, the user has to set the env variable ``foo``, which is equivalent to setting the variable ``key`` and to ``LABEL io.whalebrew.config.environment '["key"]'`` 
2. As for, ``${foobar}`` and ``${bar}=value`` the user is actually setting the name of the environment variable to be passed into the container by setting ``foobar`` and ``bar``. The image might only understand a limited set of predefined env variables passing in a new variable doesn't make sense.
3. The use case I find interesting is if there are a few env variables that requires the same value, the user can get away with setting just one. 
For example, ``LABEL io.whalebrew.config.environment '["bar=${foo}","baz=${foo}"]'``, the user can get away with setting ``foo`` instead or all the other variables.

What am I missing here? How do you use variable expansion?